helm_deploy/court-case-service/templates/_envs.tpl

```diff
@@ -193,6 +193,18 @@ env:
         name: sqs-pic-new-offender-events-dlq-secret
         key: sqs_queue_name
 
+  - name: DATABASE_READONLY_USERNAME
+    valueFrom:
+      secretKeyRef:
+        name: court-case-service-readonly-user
+        key: database_readonly_username
+
+  - name: DATABASE_READONLY_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: court-case-service-readonly-user
+        key: database_readonly_password
+
```

Comment:
I am not sure that we would want to expose this to the application itself. I appreciate it is for the purposes of running the flyway script but I can't see any further use for it. 

Have you thought about using terraform for this? It seems like it might be generally useful to all teams.

https://stackoverflow.com/questions/60820805/create-read-only-postgres-user-with-terraform is a starting point